### ISSUE TITLE
Make Add-on compatible with e10s (multiprocess)

### ISSUE THREAD

**Issue by RalfJung** [choobin/dcqs#10]
According to the compatibility reporter and https://arewee10syet.com/, this addon is not compatible with e10s (aka Firefox Multiprocess). The add-on is not at all interacting with tab contents, so it seems like it can't be too hard to establish said compatibility. That would be great, because it would mean that once e10s is rolled out for more people, this add-on won't prevent anyone from using it :)

**Comment by eddy-geek** [comment]
Yes would be nice -- and easy, as only the add-on manifest needs to be changed to add:

    <em:multiprocessCompatible>true</em:multiprocessCompatible>

Note that the current situation means people installing the add-on cannot benefit from e10s (except on Developer and Nightly channels).

**Comment by ssokolow** [comment]
Not easy anymore. Firefox 57 ([August for Developer Edition, November for Stable](https://wiki.mozilla.org/RapidRelease/Calendar)) [will remove](https://blog.mozilla.org/addons/2017/02/16/the-road-to-firefox-57-compatibility-milestones/) the ability to run XUL extensions, so this needs to be ported to WebExtensions.

To get you started, here are the links you'll probably need:
* [WebExtensions documentation overview](https://developer.mozilla.org/en-US/Add-ons/WebExtensions)
* [`commands` `manifest.json` reference](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/commands) (For the "override Ctrl+Q" part)
* [`commands` API reference](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/commands) (In case you need to hang a "do nothing" handler on it for Firefox to override the default behaviour.)

...and given that Firefox is a bit more conservative than Chrome in what it allows that API to override, you need to test it out *A.S.A.P.* in case it's necessary to go on Bugzilla and get a fix discussed, implemented, and committed before Firefox 57 moves from nightly to aurora channel in August.

**Comment by juninho** [comment]
Hi, 

There is a new addon called "Disable Ctrl-Q and Cmd-Q" that supports webextensions:
https://addons.mozilla.org/pt-BR/firefox/addon/disable-ctrl-q-and-cmd-q/?src=search

The problem is that it does not work as expected in Linux, because of the bug https://bugzilla.mozilla.org/show_bug.cgi?id=1325692.

So, for now I'm going to stick to the current addon and hope that the bug will be fixed until Firefox 57 be released...

**Comment by Dannyzen** [comment]
Friendly update, looks like https://bugzilla.mozilla.org/show_bug.cgi?id=1325692 got a "WONT FIX" which puts linux users of this extension in a particularly interesting position for FF57
